frontend/src/cards/PopulationCard.tsx

```diff
@@ -67,9 +69,23 @@ export function PopulationCard(props: PopulationCardProps) {
     Breakdowns.forFips(props.fips).andAge(onlyIncludeDecadeAgeBrackets())
   );
 
+  const queries = [raceQuery, ageQuery];
+
+  if (props.fips.isCounty()) {
+    const sviQuery = new MetricQuery(
+      "svi",
+      Breakdowns.forFips(props.fips).andAge(onlyInclude("All"))
```

Comment:
Can you file a new issue to switch this logic from using `Age` when we should really just use a new breakdown type for data that applies to the entire geography ?